etc/create_brick.py

```diff
@@ -0,0 +1,36 @@
+import etcd
+import json
+import uuid
+
+
+integration_id = "ab3b125e-4769-4071-a349-e82b380c11f4"
+job_id = str(uuid.uuid4())
+
+payload = {
+    "integration_id": integration_id,
+    "run": "tendrl.gluster.flows.CreateGlusterBrick",
+    "parameters": {
+        "GlusterBrick.brick_dict": {
+            "12.34.56.78:/mnt/tendrl_sample4/b7": {
+                "node_id": "4a47db22-b286-44da-8240-c8256b28d256",
+                "disk": "vdh"
+            },
+            "12.34.56.78:/mnt/tendrl_sample5/b8": {
```

Comment:
Given that we're allowing bricks to be created without being attached to a volume, it'll be necessary to standardise a path under which tendrl will always mount the bricks and expect bricks to be mounted, if created manually, to be able to detect them. In such a scenario, we cannot ask for a brick path, but just a name. This also ties in to the comment below against the `Create.run` method.

However, the mount path will not be available to tendrl when an existing gluster is imported. In such a case, a mount path could specified as a parameter per brick and tendrl would need to index all the mount paths, including it's own standard path, and scan all of them for syncing available bricks.